homeworks/anush_manvelyan/Maven_TestingQueue/QueueTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
```

Comment:
the same comment as for Garik...
You catch the idea but the unit tests should cover the corner cases also